composer.json

```diff
@@ -36,7 +37,6 @@
     },
     "require-dev": {
         "ext-json": "*",
-        "friendsofphp/php-cs-fixer": "^2.15",
```

Comment:
Not sure I understand the reason for removing php-cs-fixer ? We use it for most of our linting and automated lint fixes.

Reply:
I guess it conflicts with the composer packages?

Reply:
Yes, php-cs-fixer requires `"composer/semver": "^1.4"`.
https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/2.16/composer.json#L20

Reply:
It looks like `php-cs-fixer` can be re-added as it supports `composer/semver 2.x` since `v2.15.9` 👍🏻